src/NuGet.Clients/NuGet.PackageManagement.UI/NuGetPackageManagerControlSearchTask.cs

```diff
@@ -29,7 +29,7 @@ public void Start()
             {
                 await NuGetUIThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
 
-                await _packageManagerControl.SearchPackagesAndRefreshUpdateCountAsync(searchText: _searchQuery.SearchString, useCacheForUpdates: true, pSearchCallback: _searchCallback, searchTask: this);
+                await _packageManagerControl.SearchPackagesAndRefreshUpdateCountAsync(searchText: _searchQuery.SearchString, useCachedPackages: true, pSearchCallback: _searchCallback, searchTask: this);
```

Comment:
UseCachedPackages sounds like it will only use packages in our http cache...which isn't what you mean.
Perhaps -- avoidFetchingMetadata or useCachedPackageMetadata or ??

Reply:
It wasn't that specific before and wasn't confused for http cache.

Reply:
Those suggestions are fine. The point was this flag *always* was updating both Installed & Updates, so the name being specific to Updates was wrong.

Reply:
If it must be changed, I vote useCachedPackageMetadata